datacenter/dtr/2.2/guides/user/access-dtr/index.md

```diff
@@ -44,6 +44,7 @@ Then, select the following options:
 * Click 'Browser', and select 'Trusted Root Certificate Authorities'
 * Click 'Finish'
 
+[See also] (https://technet.microsoft.com/en-us/library/cc754841(v=ws.11).aspx#BKMK_addlocal).
 [Learn more about managing TLS certificates](https://technet.microsoft.com/en-us/library/cc754841(v=ws.11).aspx).
```

Comment:
There's no reason for having two links to the same place, that will confuse users. You can just update the existing url instead of adding a new `see also`.
Also, when creating urls in markdown you need to do `[]()` without a space between the two.